homeassistant/components/unifi/device_tracker.py

```diff
@@ -214,9 +207,35 @@ def __init__(self, client, controller):
         self.client = client
         self.controller = controller
 
+    @property
+    def entity_registry_enabled_default(self):
+        """Return if the entity should be enabled when first added to the entity registry."""
+        if not self.controller.option_track_clients:
+            return False
+
+        if (
+            not self.client.is_wired
+            and self.controller.option_ssid_filter
+            and self.client.essid not in self.controller.option_ssid_filter
+        ):
+            return False
+
+        if not self.controller.option_track_wired_clients and self.client.is_wired:
+            return False
+
+        return True
+
+    async def async_added_to_hass(self):
+        """Client entity created."""
+        LOGGER.debug("New UniFi client tracker %s (%s)", self.name, self.client.mac)
+
     async def async_update(self):
         """Synchronize state with controller."""
-        await self.controller.request_update()
+        if self.registry_entry.disabled_by is None:
```

Comment:
Who would call this ? As far as I see, you already guard for disabled entities in the update code